drivers/serial/uart_nrfx_uart.c

```diff
@@ -150,6 +151,11 @@ static unsigned char uart_nrfx_poll_out(struct device *dev,
 	/* send a character */
 	nrf_uart_txd_set(NRF_UART0, (u8_t)c);
 
+	/* First byte was sent, from now on relay on NRF_UART_EVENT_TXDRDY event
```

Comment:
The second part of the sentence-comment is slightly confusing: what does 

> relay on NRF_UART_EVENT_TXDRDY event status

 mean?